proxy/core/src/main/scala/io/cloudstate/proxy/valueentity/store/JdbcInMemoryStore.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudstate.proxy.valueentity.store
+
+import akka.util.ByteString
+import io.cloudstate.proxy.valueentity.store.JdbcStore.Key
+
+import scala.concurrent.Future
+
+final class JdbcInMemoryStore extends JdbcStore[Key, ByteString] {
+
+  private var store = Map.empty[Key, ByteString]
```

Comment:
Should this be a concurrent map? In Scala can use a TrieMap:

https://www.scala-lang.org/api/current/scala/collection/concurrent/TrieMap.html

Reply:
It is not intended to be used in production, I thought the concurrency level should not matter here. But I can also change it to use the more accurate TrieMap.